book.bib

```diff
@@ -8,167 +242,206 @@ @article{armstrong
   pages={097007},
   year={2019},
   publisher={Environmental Health Perspectives}
+>>>>>>> 18535dec0db0472f480e54e79895aeaf41013575
```

Comment:
die änderungen hier zeigen dir merge conflice an.
Diese bitte korrekt manuell lösen

Reply:
Also bei den Quellen hätte ich meine Quellen mit Konflikte gelöst, aber ich kann das nicht speichern. Da es heißt ich muss alle Fehler beheben, aber wie gesagt die anderen Quellen sind nicht von  mir

Reply:
ne du hast beim mergen einen Konflikt den du noch nicht gelöst hast...
Das ist das was du hier siehst:
`>>>>>>> 18535dec0db0472f480e54e79895aeaf41013575`

Alles mit den Pfeilen >>> oder <<< muss raus...

Reply:
Kann ich einfach diese Book.bib nochmal neu machen?

Reply:
Ja am einfachsten ist wsl wenn du die hier aus dem master copy pastest und NUR deine Quellen anpasst.
Hieraus: https://github.com/henrifnk/Seminar_ClimateNStatistics/blob/master/book.bib

Reply:
und in deinem files hast du auch merge Konflikte...
Dafür gilt dann das selbe...

Reply:
Ist es ein Problem, wenn ich das am Samstag mache, da ich morgen eine Klausur habe

Reply:
nein, kein problem das zählt schon zum technischen roundup dafür hast du ja nochmal eine Woche
lass dir bitte zeit dabei und mach es richtig :)!